Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -10915,6 +10915,11 @@ - (BOOL)supportsTextReplacementForWebView
     return self.webView._editable;
 }
 
+- (void)applyAutocorrection:(NSString *)correction toString:(NSString *)input shouldUnderline:(BOOL)shouldUnderline withCompletionHandler:(void (^)(UIWKAutocorrectionRects *rectsForCorrection))completionHandler
```

Comment:
Can you move this closer to the existing `applyAutocorrection:` methods?

Reply:
yes! I was looking for a better place to put it